Automattic/jetpack
# Premium Blocks: Handle not-jetpack blocks availability according to site plan.

This PR implements a way to set the availability of the block in terms of the current plan of the site. These blocks are not registered by Jetpack.

#### Changes proposed in this Pull Request:
* Set not-jetpack blocks availability according to the site plan.

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?


no

#### Testing instructions:






* Apply these changes in your testing site
* Inspect the Jetpack Initial Status and make sure that core/* blocks are available.
* Upload a video to a core/video block and make sure it works.
* Upload a video to a core/cover block and make sure it works.
* Upload an audio file to a core/audio block and make it works.

##### On WP.com
* Apply D48280-code
* Upload a video to a core/video block and make sure an upgrade banner appears that links you to the premium plan checkout.
* Upload a video to a core/cover block and make sure an upgrade banner appears that links you to the premium plan checkout.
* Upload an audio file to a core/audio block and make sure an upgrade banner appears that links you to the personal plan checkout.

#### Proposed changelog entry for your changes:

* Set not-jetpack blocks availability according to the site plan.
